### DISCUSSION TITLE
做菜翻车记--鸡鸭鱼肉篇 | /posts/fan-che/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#586] [Announcements]
https://yuanfan.rbind.io/posts/fan-che/

**Comment by yihui** [comment]
这十道菜我只做过三道。

肉确实好多，我看照片都快看饱了。

水煮鱼可以偷懒直接买调料包做。腌鱼片的时候可以加点生粉和蛋清，会更嫩一点；鱼肉容易熟，煮的时候两三分钟应该就够了。

一个月前冰的矿泉水太感人了。

烤肉的时候不确定温度高低的话，就选高，尤其是肉厚的时候。

看照片上的椒盐虾，粉壳子好像也不是很厚啊。

阿木狗居然怕鸡脖子，让我想起猫怕黄瓜。

糯米糕会蒸了就可以蒸雪媚娘了。还有冰皮月饼也差不多。

照片看起来都不错，很鼓舞士气，尤其是空气炸锅烤出来的那几样（排骨、鸡、虾），可能是因为我喜欢焦脆的缘故。

**Reply by earfanfan** [reply]
>这十道菜我只做过三道。

我猜是椒盐、水煮、三杯。我还以为这都是常见的新手进阶菜市呢，你这老厨子，感觉也没比我厉害多少啊。

>水煮鱼可以偷懒直接买调料包做。

不。等我跟你一样做菜做腻了再在调料包这件事上偷懒吧……预感没多久我就会打脸……

>腌鱼片的时候可以加点生粉和蛋清，会更嫩一点

加了蛋清，那蛋黄用来干嘛去呢？还有就是咋分离蛋清和蛋黄呢？直接拿勺子把蛋黄舀出来吗？按照我的个人经验，在打了蛋以后从那里面捞东西特别滑，以前打蛋不小心把蛋壳打进去了，要把很小一片蛋壳捞出来就很令人抓狂。

>看照片上的椒盐虾，粉壳子好像也不是很厚啊。

可是吃起来真得很像是吃土。

>阿木狗居然怕鸡脖子，让我想起猫怕黄瓜。

我今天发现它还不喜欢吃鸭屁股。我见阿木狗把鸭屁股从狗碗里叼出来，以为它是狗性发作习惯性藏食物，就又给它放回去，如此反复三次，我终于醒悟它是不想吃鸭屁股。于是我把鸭屁股扔到垃圾桶里，阿木狗还伸着狗头在垃圾桶旁边俯视了好一会鸭屁股。

>糯米糕会蒸了就可以蒸雪媚娘了。还有冰皮月饼也差不多。

雪媚娘的菜谱我查了，感觉就是各种食材按照定量的分量混合后蒸熟，不过我不大明白其中一步炒熟糯米粉是咋炒，就直接热锅倒粉下去炒么。另外我今天下单买了一个厨房秤，好像是你跟别人合开的店，四个字的店名后两个字是辉益……

我之前还用空气炸锅烤五花肉，基本做法就是先洗一些生菜、金针菇、五花肉，分别把五花肉和金针菇烤熟，撒点该撒的，然后拿生菜包着吃，那次的肉就烤得相当焦脆，因为烤得滴油了，我以为是糊了本以为只能将就吃，没想到味道还挺不错的。另，我一直以为空气炸锅很不好洗，所以以前烤东西都垫那种专门的纸，最近几次烤因为纸用完了忘了买就直接烤的，完事以后把锅用热水泡了再刷感觉很容易啊。

**Reply by earfanfan** [reply]
又另，我突然发现自从你帮忙迁移评论后，我的博客有一点变了。以前每篇博客下面都有指示前后文章的链接，那还是我从你的博客抄的，现在没了，给我赔来。

**Reply by yihui** [reply]
三道猜对了两道，三杯没做过，做过蒜香排骨。

蛋黄我一般就是煮鱼的时候一起丢进去煮了吃。小心点磕蛋，磕破了从中将蛋壳分为两半，蛋清从缝隙里流出来之后蛋黄会是完整的，留在一半蛋壳里，煮鱼的时候小心倒到锅里，煮出来还可以是完整的蛋黄。也有专门的分蛋器，就是把整个鸡蛋打到那个能让蛋清漏下去、蛋黄留在分蛋器的窝窝里，但我一般都是直接徒手分蛋。还有一种分蛋的办法是整个鸡蛋打到碗里，再用一半蛋壳把蛋黄从碗里舀出来。如果觉得单独一个蛋黄不想煮的话，也可以打三个蛋，其中一个蛋的蛋清用来腌鱼，剩下蛋黄和另外两个蛋一起做别的菜，西红柿炒蛋或韭菜鸡蛋等等。

蛋壳要是打到碗里了我一般就是徒手去捞，用勺子捞会如你所说特别滑，用筷子也慢，我就直接用手指去摸。

炒糯米粉最好用小的不粘锅，就是给糯米粉加个热而已，让它熟。可以少炒点，它并没有什么特别的食用作用，唯一的作用只是防粘，因为糯米团实在是太沾手了。

我开的秤店都被你发现了……

我至今也没怎么用过空气炸锅，家里八年前买了一个，我用了两次觉得不是很好用就收起来堆在墙角攒灰了。现在的空气炸锅似乎跟我那时候用的不太一样了，会更方便一点，不过我已经几乎没有购买新厨具的欲望了。

前后链接应犹在，只是朱颜改。

**Reply by earfanfan** [reply]
>炒糯米粉最好用小的不粘锅

为撒呢，有撒讲究么？我只有一只炒锅，我家厨房特别特别小，多买任何一只新锅子都放不下了。如果只是防粘的话，那么在蒸之前弄一点点生糯米粉似乎也行。我第二次做牛奶糯米团子的时候，就是这么干的。

你教的分出蛋清的方法在我的想象中不好实现啊，因为蛋壳破开的口子是向下的，我要看着只让蛋清掉下去而保住蛋黄的话，我的脑袋比手高很多，是不大方便去盯着那个口子的。难道你做饭收拾食材的时候都是坐着弄的？

另外，加蛋清这个方法，我在讲日本寿喜锅的视频里面看到过类似的，那里面日本人吃寿喜锅就是先在碗里磕一颗蛋的，煮好的食物夹出来都会在碗里沾点蛋液。

**Reply by yihui** [reply]
普通锅炒糯米粉的话，火小点可能也可以，不然粘在锅底的粉容易糊。这种糯米粉蒸制品一般是蒸熟了以后再揪捏成团，那时候比较沾手，可以戴一次性的防粘手套。

鸡蛋不用看，磕破了蛋清自然就从裂缝流出来了，就是注意别把裂缝掰得太大了，要让裂缝能拦住蛋黄。

**Comment by liechi** [comment]
水煮鱼的灵魂是重油和辣香。汤的温度要高，鱼片码好味后，直接滑进去，这样口感才够嫩；最后撒辣椒，淋油上桌。吃不了重口的话，做酸菜鱼比较好。

照烧鸡腿我一般是去超市买现成的，没有自己做过；看起来很香，但吃起来有点腻。等以后有空了可以挑战一下，看看能不能做得比超市的好点。

鸭头都不吃，看来是无福消受兔头了。不过鸭头本来也没什么肉，稍微可吃的是把头骨咬开后里面的脑髓---估计你是更吃不了了。三杯杏鲍菇味道和口感都不错，多加辣，可以当零食吃。

**Reply by earfanfan** [reply]
>鱼片码好味后，直接滑进去

这跟益辉讲拿生粉和蛋清腌一下是一个意思么？滑进去是什么操作，我的锅侧面和底部呈九十度，又不是像滑滑梯那样倾斜，鱼片还能怎么滑啊。

>看起来很香，但吃起来有点腻

我也做不出来那种看起来香的效果。肉多吃几口都是会腻的，搭配蔬菜水果就还好。

>稍微可吃的是把头骨咬开后里面的脑髓

我把鸭子分尸的时候，第一步就是把头砍下来，砍头的时候有摸到鸭头下巴是有肉的，但万万没想到是要吃脑髓……既然吃鸭脑髓这么费事，那么干嘛不直接把脑髓取出来再做着吃咧，下次我再吃鸭子的话试试把鸭脑袋剁开看看脑髓长撒样。

另外，用嘴把头骨咬开这件事，我肯定也办不到，最多敢用手从鸭子的扁嘴那里暴力掰开。各种食材的头我都不敢吃，因为它们都有眼睛啊。

**Reply by liechi** [reply]
>这跟益辉讲拿生粉和蛋清腌一下是一个意思么？

对。

>滑进去是什么操作，我的锅侧面和底部呈九十度，又不是像滑滑梯那样倾斜，鱼片还能怎么滑啊。

放进去就行，说滑主要是描述这个过程要轻，别把热汤溅起来烫着自己。

>那么干嘛不直接把脑髓取出来再做着吃咧

这个过程可能跟嗑瓜子一样，让别人剥好了直接吃仁就没那么香了。

>各种食材的头我都不敢吃，因为它们都有眼睛啊。

最近第二次听到这个话。
